spec/rubocop/cop/layout/closing_parenthesis_indentation_spec.rb

```diff
@@ -94,10 +84,10 @@
             some_method(a,
               x: 1,
               y: 2
-                       )
+                        )
             b =
               some_method(a,
-                         )
+                          )
```

Comment:
Same here.

Reply:
I was just making it more explicit that the paren wasn't aligned.